Remove obsolete documentation on json-api v1 (#893)
digital-asset/docs.daml.com: docs/3.1/docs/json-api/v1/index.rst

    daml new my-project --template quickstart-java
    cd my-project
    daml build
    daml sandbox --json-api-port 8080 --wall-clock-time --dar ./.daml/dist/quickstart-0.0.1.dar

.. _start-http-service:

Configure the HTTP JSON API Service
===================================

> You can configure the JSON API through the standard canton config file:

Reviewer: ```suggestion
You can configure the JSON API through the standard Canton config file:
```

.. code-block:: shell

    daml sandbox -c canton.conf --wall-clock-time --dar ./.daml/dist/quickstart-0.0.1.dar

where a corresponding minimal config file is

.. code-block:: none

    canton.participants.sandbox {
      http-ledger-api {
        server {
          address=localhost
          port=7575
        }
      }
    }

> This will start the JSON API on port 7575.

Reviewer: ```suggestion
This starts the JSON API on port 7575.
```

        mode = "shaping"
      }

      // DEV MODE ONLY (not recommended for production)
      // Allow connections without a reverse proxy providing HTTPS.
      allow-insecure-tokens = false

      // Turn on logging of http requests and responses
      debug-logging-of-http-bodies = false

      // utilize the user management database (including idp) to convert JWT tokens to parties
      // that the user is entitled to act and read as even if the authorization is not turned on.
      user-management-without-authorization = false
    }

With Query Store
----------------

Unlike the previous stand-alone JSON API, the built-in version does not have a query store. If you need persistent

> storage for your JSON API client application, please configure a PQS instance.

Reviewer 2: A link to docs/query/pqs-user-guide would be good

Author: done

        mode = "shaping"
      }

      // DEV MODE ONLY (not recommended for production)
      // Allow connections without a reverse proxy providing HTTPS.
      allow-insecure-tokens = false

      // Turn on logging of http requests and responses
      debug-logging-of-http-bodies = false

      // utilize the user management database (including idp) to convert JWT tokens to parties
      // that the user is entitled to act and read as even if the authorization is not turned on.
      user-management-without-authorization = false
    }

With Query Store
----------------

Unlike the previous stand-alone JSON API, the built-in version does not have a query store. If you need persistent

> storage for your JSON API client application, please configure a PQS instance. For setup instructions, see the `PQS

Reviewer: ```suggestion
storage for your JSON API client application, configure a PQS instance. For setup instructions, see the `PQS
```
